src/wallet/wallet.h

```diff
@@ -1390,8 +1390,13 @@ class WalletRescanReserver
 * Use DummySignatureCreator, which inserts 71 byte signatures everywhere.
 * NOTE: this requires that all inputs must be in mapWallet (eg the tx should
 * be IsAllFromMe). */
-std::pair<int64_t, int64_t> CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, bool use_max_sig = false) EXCLUSIVE_LOCKS_REQUIRED(wallet->cs_wallet);
-std::pair<int64_t, int64_t> CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, const std::vector<CTxOut>& txouts, bool use_max_sig = false);
+struct TxSize {
+    int64_t vsize;
+    int64_t weight;
```

Comment:
```suggestion
    const int64_t vsize{-1};
    const int64_t weight{-1};
```

Reply:
const makes it unhappy due to default copy constructor being ill-formed, removed